docker-compose.yml

```diff
@@ -11,17 +11,62 @@ services:
             - pg_data:/var/lib/postgresql/data
         ports:
             - "${PG_PORT}:5432"
-        
+    db_exporter:
+        image: quay.io/prometheuscommunity/postgres-exporter
+        environment:
+            - DATA_SOURCE_NAME=postgresql://isb_writer:isamplesinabox@db:5432/postgres?sslmode=disable
+        expose:
+            - "9187"
+        depends_on:
+            - db
+
     solr:
         build: ./solr/
         volumes:
             - solr_data:/var/solr
         ports:
-            - "${SOLR_PORT}:8983"            
+            - "${SOLR_PORT}:8983"
+        expose:
+            - "9983"
```

Comment:
The `9983` is the zookeeper port.  Open it up so the `solr_exporter` can get all the stats it needs.